axelrod/strategies/mindreader.py

```diff
@@ -22,7 +22,7 @@ class MindReader(Player):
 
     name = 'Mind Reader'
     classifier = {
-        'memory_depth': -10,
```

Comment:
Same

Reply:
Yeah I thought it was probably due to the "looking in to the future" but taking the notion of infinity to mean "backwards and forwards" I think we're good :) (I agree that we don't care too much :)).